libs/dmlf/include/dmlf/distributed_learning/distributed_learning_client.hpp

```diff
@@ -63,37 +48,35 @@ class TrainingClient
   using VectorTensorType = std::vector<TensorType>;
   using TimestampType    = int64_t;
   using GradientType     = fetch::dmlf::Update<TensorType>;
-  using GraphPtrType     = std::shared_ptr<fetch::ml::Graph<TensorType>>;
+  using ModelPtrType     = std::shared_ptr<fetch::ml::model::Sequential<TensorType>>;
 
 public:
   TrainingClient(std::string id, ClientParams<DataType> const &client_params,
                  std::shared_ptr<std::mutex> console_mutex_ptr);
 
-  TrainingClient(
-      std::string id, GraphPtrType graph_ptr,
-      std::shared_ptr<fetch::ml::dataloaders::DataLoader<TensorType, TensorType>> loader_ptr,
-      std::shared_ptr<fetch::ml::optimisers::Optimiser<TensorType>>               optimiser_ptr,
-      ClientParams<DataType> const &client_params, std::shared_ptr<std::mutex> console_mutex_ptr);
+  TrainingClient(std::string id, ModelPtrType model_ptr,
+                 ClientParams<DataType> const &client_params,
+                 std::shared_ptr<std::mutex>   console_mutex_ptr);
 
   virtual ~TrainingClient() = default;
 
-  void SetNetworker(std::shared_ptr<fetch::dmlf::AbstractLearnerNetworker> i_learner_ptr);
+  void SetNetworker(std::shared_ptr<fetch::dmlf::AbstractLearnerNetworker> networker_ptr);
 
   virtual void Run();
 
   void Train();
 
   virtual void Test();
 
-  virtual std::shared_ptr<GradientType> GetGradients();
+  virtual std::shared_ptr<fetch::dmlf::Update<TensorType>> GetGradients();
```

Comment:
Should be GradientType? Or remove `using GradientType     = fetch::dmlf::Update<TensorType>;` from above.

Reply:
oh I didn't know that using existed. That's an extremely confusing using because an Update and a gradient are fundamentally different things; i'll rename it.